dev/io.openliberty.checkpoint_fat/bnd.bnd

```diff
@@ -59,7 +61,8 @@ tested.features: \
 	ejb-2.3,\
 	jsf-2.3,\
 	ssl-1.0,\
-	transportSecurity-1.0
+	transportSecurity-1.0,\
+	user.feature.checkpoint.bells-1.0
```

Comment:
I would only expect to list things here if the features are enabled programmatically.  For example: `config.getFeatureManager().getFeatures().add("osgiConsole-1.0");`